src/server/game/Movement/Spline/MovementPacketBuilder.cpp

```diff
@@ -191,7 +191,10 @@ namespace Movement
 
             uint32 nodes = move_spline.getPath().size();
             data << nodes;
-            data.append<Vector3>(&move_spline.getPath()[0], nodes);
+            if (nodes > 0)
```

Comment:
this can't be if (nodes) ?

Reply:
Of course it can be.

Reply:
so why not?

Reply:
Really? It changes anything?

Reply:
ofc it doesnt change anything, but as far as I can see and remember that is how most of such checks are made. No reason to provide extra things that doesn't do anything?

Reply:
I'd keep it `nodes > 0`

Reply:
It's an uint32 so it will never be < 0

Reply:
But it can be 0...

Reply:
I’m not sure if I am following. If it’s 0, it will return false with the changes @Kitzunu suggested. Yes, it’s the same as it does currently, but the other approach is more readable as the value can never be negative, due to unsigned variable

Reply:
I'll change it, it's pointless to have the PR waiting for this.